quickget

```diff
@@ -352,6 +354,15 @@ function releases_ubuntu() {
     canary
 }
 
+function releases_void() {
+    echo i686-20210930 \
```

Comment:
I'd suggest not to include other releases while the get() only has one hard-coded option. Maybe also consider using the provided hash file to derive the latest iso for $RELEASE having given a choice of "cli or xfce" and optionally also "musl" flavour or not - examples exist to re-use (and I might send you a PR if time allows - no promises mind)

Reply:
repaired